frontend/stylesheets/main.less

```diff
@@ -456,41 +457,3 @@ img.img-50 {
 .link-in-progress {
   cursor: wait;
 }
-
-.keyboard-key-icon {
-  border: 2px solid #888;
-  border-radius: 2px;
-  min-width: 26px;
-  min-height: 26px;
-  margin: 4px;
-  display: inline-block;
-  text-align: center;
-  font-size: 10px;
-  vertical-align: middle;
-  line-height: 24px;
-  font-weight: bold;
-}
-
-.keyboard-key-icon-small {
-  border: 1px solid rgb(255, 255, 255);
-  border-radius: 2px;
-  min-width: 14px;
```

Comment:
This css is only used in the tracing view. Thus I moved it to the match less file